rebasehelper/output_tool.py

```diff
@@ -115,9 +114,12 @@ def print_patches_section_cli(cls, color, patch_type):
             return
 
         if patch_type in patches:
-            print('\n%s patches:' % patch_type)
+            logger_output.info('\n%s patches:', patch_type)
             for patch in patches[patch_type]:
-                ConsoleHelper.cprint(patch, color=color)
+                if color == 'green':
```

Comment:
No, this is wrong. `BaseOutputTool.print_patches_cli()` should be changed to map to logging levels instead of colors.